CHANGELOG.rst

```diff
@@ -0,0 +1,40 @@
+=============
+RTV Changelog
+=============
+
+.. _1.2.1: http://github.com/michael-lazar/rtv/releases/tag/v1.2.1
+.. _1.2: http://github.com/michael-lazar/rtv/releases/tag/v1.2
+
+------
+1.2.1_
```

Comment:
Can you add dates to the release numbers? Besides that, this looks damn good :+1: